<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSchoolManager</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tbPupil">
       <attribute name="title">
        <string>Люди</string>
       </attribute>
       <attribute name="toolTip">
        <string>Ученики, классы, учителя</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0" colspan="2">
         <widget class="QFrame" name="boxClass">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QPushButton" name="cmdAdd">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="cmdOpenCard">
             <property name="text">
              <string>Открыть
карточку</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="cmdRemove">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="cmdCloseClass">
             <property name="statusTip">
              <string>Закрыть класс</string>
             </property>
             <property name="text">
              <string>Закрыть</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="1" rowspan="2">
            <widget class="QPushButton" name="cmdCloseRemove">
             <property name="text">
              <string>Закрыть
и удалить
класс</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblTitle">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Нажмите правой кнопкой мыши, чтобы изменить....&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="cmdMove">
             <property name="text">
              <string>Переместить</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" rowspan="11">
            <widget class="QListWidget" name="lstPupils"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QFrame" name="taskBox">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QComboBox" name="cbLetter"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="cmdOpenClass">
             <property name="text">
              <string>Открыть</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+O</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="cbClass"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="cmdCreateClass">
             <property name="text">
              <string>Создать класс</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tbBooks">
       <attribute name="title">
        <string>Книги</string>
       </attribute>
       <attribute name="toolTip">
        <string>Книги, учебники</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="mnuQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Класс</string>
    </property>
    <addaction name="actCreateClass"/>
    <addaction name="actOpenClass"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuQuit">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
   <property name="statusTip">
    <string>Выход из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actOpenClass">
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
  </action>
  <action name="actCreateClass">
   <property name="text">
    <string>&amp;Создать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
